cypress/integration/backend-admin-ui/test_2_dataset.js

```diff
@@ -61,20 +61,21 @@ context("Admin UI Single Dataset", () => {
         });
     });
 
-    describe("Can delete test table and concept", () => {
-
+    describe("Can delete test concept", () => {
         it("Can delete test concept", () => {
             cy.get('[data-test-id="accordion-Concepts"]').click();
             cy.get(`[data-test-id="delete-btn-concept-${testDSID}.concept1"]`).click({force: true});
             cy.get(`[data-test-id="delete-btn-${testDSID}.concept1"]`).should('not.exist');
         });
+    });
 
+    describe("Can delete test table", () => {
         it("Can delete test table", () => {
+            cy.wait(100);
```

Comment:
Does this wait decides whether the test passes or not?
It looks tricky.
If possible i propose to call `visitAdminUI(`datasets/${testDSID}`);` instead

Reply:
It should make a difference. Locally there is no change if the wait is set or not, it always passes.
We thought about using `visitAdminUI` but I was unsure, if it actually reloads the page if the same url is given.
But after reading the documentation (https://docs.cypress.io/api/commands/visit) it seems like it should always reload.
I will change it to `visitAdminUI`.